docs/navigation.rb

```diff
@@ -138,8 +138,8 @@ def without_prefix(prefix)
             path: 'Hashes-and-Password-Cracking.md'
           },
           {
-            path: 'msfdb:-Database-Features-&-How-to-Set-up-a-Database-for-Metasploit.md',
-            new_base_name: 'Metasploit-Database-Support.md',
+            old_wiki_path: 'msfdb:-Database-Features-&-How-to-Set-up-a-Database-for-Metasploit.md',
```

Comment:
This old path isn't really required, but leaving the metadata around incase it's needed for future automated changes to the old wiki seems sensical